lib/formatters/ecto/changeset.ex

```diff
@@ -1,8 +1,24 @@
 defmodule DeliriumTremex.Formatters.Ecto.Changeset do
+  def format({key, [%{} | _] = errors} = _error) when is_list(errors) do
+    errors = Enum.filter(errors, fn x -> not (x == %{}) end)
```

Comment:
Is the purpose of this line to check that the map is not empty or to check that X is not a map?

Reply:
I think that she meant to check if map is empty. There is a function in Elixir Kernel `map_size/1` which does the job, and it's much more descriptive that this anon function.

Reply:
It removes empty maps from list. I have extracted into a separate function.